### PR TITLE
Add an introduction in the readme, remove/update it

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,108 @@
...
-Every check can be disabled with the following config (the configuration is `ci/config.yaml`):
+## Publishing
 
-```yaml
-checks:
-  <check name>: false
-```
+The main goals of C2C CI utils is to offer the commands and the workflows to have the following project structure:
+
+Have stabilization branch named by default `<major>.<minor>`.
```
Comment (vuilleumierc) [inline README.md:7]: I guess we can have multiple stabilization branches?
```suggestion
Have stabilization branches named by default `<major>.<minor>`.
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,108 @@
...
+Have od the release named by default `<major>.<minor>-<patch>`.
```
Comment (vuilleumierc) [inline README.md:8]: Not sure about "od" - maybe a typo?

File: README.md [hunk 3]
```diff
@@ -1,58 +1,108 @@
...
+
+With C2C CI utils you can publish a python package and a Docker image from the same repository.
+
+The default publishing are:
+
+- Push on the `<major>.<minor>` branch will publishing Docker images (the version in the last line af the
```
Comment (vuilleumierc) [inline README.md:14]:
```suggestion
- Push on the `<major>.<minor>` branch will publish Docker images (the version in the last line of the
```

File: README.md [hunk 4]
```diff
@@ -1,58 +1,108 @@
...
+  `SECURITY.md` of the `master` branch will be published also using the `latest` tag).
+- Create the tag `<major>.<minor>.<patch>` will publish the Docker images, and the Python package.
+- Push on a feature branch (whatever other name) will publish the Docker images.
+- Delete a feature branch will delete the Docker images.
+- Push on the `master` branch will publish the Docker images with the master tag (Publishing a python package is also possible).
+
+The Docker images are published on Docker Hub and GitHub Container Registry.
+
+.. Note::
```
Comment (vuilleumierc) [inline README.md:23]: I think there is a formatting problem here?

File: README.md [hunk 5]
```diff
@@ -1,58 +1,108 @@
...
+
+    Dry run publish: `GITHUB_REF=... c2cciutils-publish --dry-run ...`
+
+## Changelog
+
+Then we create a tag by default with the `changlog` workflow a release is created on GitHub, a changelog is
```
Comment (vuilleumierc) [inline README.md:29]:
```suggestion
When we create a tag by default with the `changelog` workflow a release is created on GitHub, a changelog is
```

File: README.md [hunk 6]
```diff
@@ -1,58 +1,108 @@
...
+generated add added to the release.
```
Comment (vuilleumierc) [inline README.md:30]:
```suggestion
generated and added to the release.
```

File: README.md [hunk 7]
```diff
@@ -1,58 +1,108 @@
...
+adding a label named `backport <destination_branch>` on the pull request.
+
+## Old workflows
+
+GitHub will retain all the old workflows, so we need to delete them, the `delete-old-workflows-run`
+workflow will delete the workflows older than 500 days.
+
+## Workflows
 
 It make easier to place the following workflows:
```
Comment (vuilleumierc) [inline README.md:73]: The sentence is not clear to me

**Review by vuilleumierc — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -1,58 +1,108 @@
...
+C2C CI utils will no mere provide a tool to do a check of the project, this is replaced by `pre-commit`,
+a base configuration is provided in the example project.
+
+## Pull request checks
+
+A workflow is provided to run the checks on the pull requests, it will run the `c2cciutils-pr-checks` command.
+
+- Check that the commit message and the pull request title start with a capital letter.
+- Check that there aren't any spelling issue in the commit message and in the pull request title.
+- Add a message to the pull request to the JIRA issue if the pull request branch name start with
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply README.md:52]: For me, it's clear, I spear about this kind of messages: https://github.com/camptocamp/tilecloud/pull/1548#issuecomment-1609391111

File: README.md [hunk 9]
```diff
@@ -68,12 +118,6 @@ In the CI we need to have the following secrets::
   the secrets exists in the Camptocamp organization but not shared on all project, then you should add
   your project to the shared list.
 
-## Use locally, in the projects that use c2cciutils
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply README.md:71]: Back to the previous content without the deprecated `c2cciutils-checks` for the new project there is the relay shirt section `New project`...

File: README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (vuilleumierc) [reply README.md:52]: OK I see, then I would rather write something like:
```suggestion
- Add a message to the pull request with the JIRA issue if the pull request branch name start with
```

File: README.md (hunk 4 above)
Comment (sbrunner) [reply README.md]: Fixed, thanks :-)

**Pull request merged**